### ISSUE TITLE
Error On Page "HTML-CSS-JS-Webpack-Starter" At:  "About"  /  Error en la página "HTML-CSS-JS-Webpack-Starter" En: "Acerca De"

### ISSUE THREAD

**Issue by anaPiris** [Azordev/various-tasks#129]
**Explanation:**
In "About" you do not have the link that explains the content of the page.

**Current result:**
When entering the "HTML-CSS-JS-Webpack-starter" page, in "About" there is no link that explains the content of the page.

**Expected result:**
That when entering the "HTML-CSS-JS-Webpack-starter" page, in "About" it shows the link that explains the content of the page.

**Screen size:**
Full Screen.
screenshots:
![image](https://user-images.githubusercontent.com/74306449/167219275-392aec18-7350-4c8e-8083-5b1bf96e1f10.png)

**Check list:**
This issue can be closed when the following tasks are completed:
-That the indicated error be modified.

**References:
https://github.com/Israel-Laguan/HTML-CSS-JS-Webpack-starter

-----------------------------------------------------------------

**Spanish Version**

**Explicación:**
En "Acerca de" no tienes el enlace que explica el contenido de la página.

resultado actual:
Al ingresar a la página "HTML-CSS-JS-Webpack-starter", en "Acerca de" no hay ningún enlace que explique el contenido de la página.

Resultado Esperado:
Que al ingresar a la página "HTML-CSS-JS-Webpack-starter", en "Acerca de" muestra el link que explica el contenido de la página.

Tamaño de pantalla:
Pantalla completa.
capturas de pantalla:
![imagen](https://user-images.githubusercontent.com/74306449/167219275-392aec18-7350-4c8e-8083-5b1bf96e1f10.png)

Lista de Verificación:
Este problema se puede cerrar cuando se completen las siguientes tareas:
-Que se modifique el error indicado.

Referencias:
https://github.com/Israel-Laguan/HTML-CSS-JS-Webpack-starter

**Comment by Israel-Laguan** [comment]
Same issue as #121 @angelik0828 please verify

**Comment by angelik0828** [comment]
yes, it is the same error, but there is a difference in the link where it is being seen, it is a different repository

![image](https://user-images.githubusercontent.com/4030477/167402706-5f9fbd92-3593-4ce7-96c2-db4f10568cb7.png)
![image](https://user-images.githubusercontent.com/4030477/167402730-d9b1165c-2612-4e50-a4f4-b358d8ff4b1f.png)
